app/controllers/patients_controller.rb

```diff
@@ -19,7 +19,7 @@ def show
     @patient = current_user.get_patient(params.permit(:id)[:id])
 
     # If we failed to find a subject given the id, redirect to index
-    redirect_to(root_url) && return if @patient.nil?
+    redirect_to(action: 'monitoree_unavailable') && return if @patient.nil?
```

Comment:
The page no longer shows the ID with the route change I requested. I should have mentioned, but you will also need to do this change along with it.
```suggestion
    redirect_to(action: 'monitoree_unavailable', id: params[:id]) && return if @patient.nil?
```

Reply:
good catch, updated here: https://github.com/SaraAlert/SaraAlert/pull/736/commits/0c3bf5e8cc8e403083ccb5500bc1ece33dea2d86

Reply:
got catch: https://github.com/SaraAlert/SaraAlert/pull/736/commits/24eb9e95f3781d1067cbb86d3926b45d6e3fc1c0